rs/consensus/tests/framework/mod.rs

```diff
@@ -70,7 +70,7 @@ pub fn setup_subnet<R: Rng + CryptoRng>(
                 .iter()
                 .map(|key_id| KeyConfig {
                     key_id: key_id.clone(),
-                    pre_signatures_to_create_in_advance: 4,
+                    pre_signatures_to_create_in_advance: if key_id.is_idkg_key() { 4 } else { 0 },
```

Comment:
How about
```
if let MasterPublicKeyId::VetKd(_) = key_id { 0 } else { 4 }
```
instead to make this more robust in case we add another non-iDKG chain key variant that does have pre-signatures?